packages/dev-server/package.json

```diff
@@ -17,10 +17,14 @@
   ],
   "author": "",
   "license": "MIT",
-  "devDependencies": {
+  "gitHead": "b50f7a7fca617964ade5a3ecb5225fee17b20fe4",
+  "dependencies": {
+    "chalk": "^2.4.2",
+    "compression": "^1.7.4",
+    "express": "^4.17.1",
     "redux-logger": "^3.0.6",
+    "webpack": "^4.33.0",
     "webpack-dev-middleware": "^3.6.1",
     "webpack-hot-middleware": "^2.22.0"
-  },
-  "gitHead": "b50f7a7fca617964ade5a3ecb5225fee17b20fe4"
+  }
```

Comment:
```suggestion
  },
  "gitHead": "b50f7a7fca617964ade5a3ecb5225fee17b20fe4"
```

Reply:
It looks like it's still there in line 20. Sort of weird how the diff is showing it, not sure why that's happening. Let me know if I'm missing something though

Reply:
Ah, strange! Thanks for checking!